<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Read It Aloud</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>239</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="readButton">
       <property name="text">
        <string>Read</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="text">
        <string>Pause</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="text">
        <string>Stop</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>761</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QSlider" name="volumeSlider">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="resetVolumeButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QSlider" name="pitchSlider">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="resetRateButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="browseButton">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="localeLabel">
       <property name="text">
        <string>Locale:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QSlider" name="rateSlider">
       <property name="minimumSize">
        <size>
         <width>84</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="volumeLabel">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="rateLabel">
       <property name="text">
        <string>Rate:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="resetPitchButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="localesCombo"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fileLabel">
       <property name="text">
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="lineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="voiceLabel">
       <property name="text">
        <string>Voice:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="pitchLabel">
       <property name="text">
        <string>Pitch:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QComboBox" name="voicesCombo"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
